controlpanel/api/slack.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+import slack
+
+
+CREATE_SUPERUSER_MESSAGE = "`{username}` was created as a superuser"
+GRANT_SUPERUSER_ACCESS_MESSAGE = "`{username}` was granted superuser status"
+
+
+def notify_team(message, request_user=None):
+    token = settings.SLACK['api_token']
+    if token:
+        if request_user:
+            message = f"{message} by `{request_user.username}`"
+        slack.WebClient(token=token).chat_postMessage(
+            as_user=False,
+            username=f"Control Panel [{settings.ENV}]",
+            channel=settings.SLACK["channel"],
+            text=message,
+        )
```

Comment:
So potentially the Slack `token` could be blank/not set but we don't want to blow up.
Maybe send a message is still worth and it may also help to document this design decision in the best scenario...
...and worst case scenario if we were expecting a notification in Slack and it didn't happen the logs would tell us why.

Reply:
The problem is that if the settings module raises an exception getting the token, then tests cannot run. I realise now that this does not work either - I have a better solution coming.

Reply:
Yeah I get that you may not want to send the notification going during tests etc...having a warning in the logs seems sensible.

Awesome about the improved solution ☺️